SwitchSpeak-iOS/SwitchSpeak-iOS/TouchGrid/TouchGrid.swift

```diff
@@ -68,10 +68,13 @@ class TouchGrid {
     func selectSubTree() {
         curNode.childNodes[childNumber].highlightSubTree()
         let previousCurNode: Node = curNode
-        
-        delay(self.settings.scanSpeed.rawValue) {
+		var timeDelay = 0.0
+		if (!curNode.childNodes[(childNumber) % curNode.childNodes.count].dummy) {
```

Comment:
You know by line 69 that 
`childNode % curNode.childNodes.count == childNode`
because you just wrote `curNode.childNodes[childNumber]`

Reply:
Initially the code was a little different hence needed the modulo operation. But you are right we don't need it now. Will update this change.